src/gausskernel/storage/access/k2/k2cat_cmds.cpp

```diff
@@ -269,22 +269,12 @@ K2PgCreateTable(CreateStmt *stmt, char relkind, TupleDesc desc, Oid relationId,
 void
 K2PgDropTable(Oid relationId)
 {
-	K2PgStatement	handle = NULL;
-
 	/* Drop the table */
-	{
-		bool not_found = false;
-		HandleK2PgStatusIgnoreNotFound(PgGate_NewDropTable(u_sess->proc_cxt.MyDatabaseId,
-																									 relationId,
-																									 false, /* if_exists */
-																									 &handle),
-																 &not_found);
-		const bool valid_handle = !not_found;
-		if (valid_handle)
-		{
-			HandleK2PgStatusIgnoreNotFound(PgGate_ExecDropTable(handle), &not_found);
-		}
-	}
+    bool not_found = false;
+    HandleK2PgStatusIgnoreNotFound(PgGate_ExecDropTable(u_sess->proc_cxt.MyDatabaseId,
```

Comment:
remove not_found, just use HandleK2PgStatus() instead of HandleK2PgStatusIgnoreNotFound()

Reply:
Done